PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalLineItemKind.kt

```diff
@@ -0,0 +1,12 @@
+package com.braintreepayments.api.paypal
+
+/**
+ * The type of PayPal line item.
+ *
+ * [KIND_CREDIT] A line item that is a credit.
+ * [KIND_DEBIT] A line item that debits.
+ */
+enum class PayPalLineItemKind(val stringValue: String) {
+    KIND_CREDIT("credit"),
+    KIND_DEBIT("debit")
+}
```

Comment:
Is duplicating the word "kind" generally a best practice in Kotlin/Java or could we drop it from either the class name or enum name?